### ISSUE TITLE
Coordinate with otf-fea-rs?

### ISSUE THREAD

**Issue by twardoch** [RazrFalcon/rustybuzz#9]
https://github.com/wrl/otf-fea-rs/ is a Rust-based compiler of FEA into GSUB/GPOS, while https://github.com/RazrFalcon/rustybuzz is a Rust-based OT shaper. 

Perhaps @wrl and @RazrFalcon could somehow coordinate so that the two libraries can interoperate. No specific idea yet but I can well imagine this to be beneficial.

**Comment by RazrFalcon** [comment]
Well, rustybuzz is just a harfbuzz port. So I'm doing the same work harfbuzz does without much changes (except parsing itself).

**Comment by twardoch** [comment]
Right! I probably meant mostly the other way around. otf-fea-rs needs some font libraries (not sure how it works), and it's a relatively young project. I think the goal is to allow instant rebuilding of features from the text source.

I can imagine some benefits of using both, though I cannot say right away what they are :)

**Comment by twardoch** [comment]
Here's a use case that shows why, in principle, it mages sense to combine a OpenType Layout engine with an OpenType feature compiler: https://github.com/simoncozens/namvezvez/blob/master/README.md

**Comment by RazrFalcon** [comment]
It's great to see alternative implementations, but that one supports like 1% of what harfbuzz do. Also, shaping is a very performance-sensitive operation. Python will be slow as hell.
